code/_onclick/item_attack.dm

```diff
@@ -91,10 +91,10 @@ avoid code duplication. This includes items that may sometimes act as a standard
 	var/power = force
 	if(HULK in user.mutations)
 		power *= 2
-	if(istype(user, /mob/living/carbon/human))
-		var/mob/living/carbon/human/X = user
-		if(X.gloves && istype(X.gloves,/obj/item/clothing/gloves/force))
-			var/obj/item/clothing/gloves/force/G = X.gloves
-			power *= G.amplification
+	//if(istype(user, /mob/living/carbon/human)) //Removed.
```

Comment:
It should be removed properly and not just commented out.
If someone wants to restore it, they can just look up how it was previously (thanks to git)